addons/sale/views/utm_campaign_views.xml

```diff
@@ -6,16 +6,27 @@
         <field name="model">utm.campaign</field>
         <field name="inherit_id" ref="utm.utm_campaign_view_kanban"/>
         <field name="arch" type="xml">
-            <xpath expr="//div[@id='utm_statistics']" position="inside">
-                <div class="me-3" title="Revenues" groups="sales_team.group_sale_salesman">
-                    <field name="currency_id" invisible="True"/>
-                    <small class="fw-bold">
+            <xpath expr="//div[hasclass('oe_kanban_bottom_left')]" position="inside">
+                <a t-if="record.invoiced_amount" href="#"
+                    groups="sales_team.group_sale_salesman"
+                    data-type="object" data-name="action_redirect_to_invoiced"
+                    class="oe_kanban_action oe_kanban_action_a oe_kanban_badge_pill me-1 order-1"
+                    style="margin-bottom: 1px;" string="Revenues">
```

Comment:
Ideally we should not have inline style in views, is that 1 pixel actually necessary? ^^